WORKSPACE

```diff
@@ -65,11 +65,18 @@ rbe_autoconfig(
 )
 
 load("//kotlin:kotlin.bzl", "kotlin_repositories", "kt_register_toolchains")
-
 kotlin_repositories()
-
 kt_register_toolchains()
 
+http_archive(
```

Comment:
There's a vendored @rules_java in every Bazel installation. Do we need this here?

Reply:
Oh? Nice! I guess we do not. Though it semi-forces the kotlin rules to use whatever is in the current bazel... that definitely ties the lifecycles, but in practice, they're already tied, so it's no worse, and there's less hassle. I'll remove. Do I need to declare it or is it just available as `@rules_java`?

Reply:
It's available as @rules_java, no need to declare. Bazel prefixes and suffixes every WORKSPACE with some implicit repositories, and @rules_java is one of them.

Reply:
Nice - is there a list? (Also... done)

Reply:
here, https://source.bazel.build/bazel/+/master:src/main/java/com/google/devtools/build/lib/bazel/rules/java/jdk.WORKSPACE;l=3;drc=a6cd408876dd55c988a123af8237807bee549a5b has some but not all. 

There's no canonical list AFAIK. you can also get a hint by looking at Bazel's own WORKSPACE file here:

https://source.bazel.build/bazel/+/master:WORKSPACE;l=169-233;drc=168d0a7aa2af7c99a7ed2cd98b89fceb73aebd48